presidio-anonymizer/tests/integration/test_engine.py

```diff
@@ -64,7 +106,7 @@ def test_given_anonymize_with_encrypt_then_text_returned_with_encrypted_content(
 
     actual_anonymize_result = AnonymizerEngine().anonymize(
         text, analyzer_results, anonymizers_config
-    )
+    ).text
```

Comment:
Instead of doing .text consider checking the indices here as well.
Same for the other tests.